app/tests/service/ocr.spec.ts

```diff
@@ -0,0 +1,50 @@
+import ocr from "@/service/ocr";
+import path from "path";
+import { afterEach, describe, expect, it, vi } from "vitest";
+
+const { logger, scanImage } = ocr;
+const { parseEarningsStatement, parseW2Data } = ocr.parsers;
+
+describe("./src/utils/ocr/index.ts - OCR", () => {
+  afterEach(() => {
+    vi.resetAllMocks();
+  });
+
+  describe("parseEarningsStatement", () => {
+    describe("when the earnings statement is valid", () => {
```

Comment:
Notice that within the test suite descriptor we have a happy path and a path that's [unfavorable](https://github.com/DSACMS/iv-doc-uploader/pull/7/files#diff-a8d18c83dccf52dbe3a37ffcc1712133dbb071d7a03a1b50f964d941ec81b9fcR38).